assets/large_language_models/rag/components/src/flow_creation.py

```diff
@@ -135,10 +136,15 @@ def get_connection_name(s):
 
 def get_deployment_and_model_name(s):
     """get_deployment_and_model_name."""
-    deployment_and_model = s.split("/deployment/")[1]
-    deployment_name = deployment_and_model.split("/model/")[0]
-    model_name = deployment_and_model.split("/model/")[1]
-    return (deployment_name, model_name)
+    model_type = s.split("://")[0]
```

Comment:
⁠I wonder if this should be helper function in a util folder (or azureml-rag) and used across multiple components.
FYI- side note- do other components support this?

Reply:
I agree this could work well as a helper function in azureml-rag. The embeddings and QA Gen component do support open ai models, I made the changes a while back